setup.py

```diff
@@ -111,7 +111,7 @@ def build_manpages():
             'requests_oauthlib',
             'reflink',
             'rarfile',
-            'python3-discogs-client',
+            'python3-discogs-client>=2.3.15',
```

Comment:
Made the requested changes and also added a minimum version for the `extras_require 'test'`. Not sure why there wasn't one before, but if there's a reason I'm happy to revert.